raiden/raiden_service.py

```diff
@@ -19,8 +19,8 @@
 
 log = slogging.get_logger(__name__)  # pylint: disable=invalid-name
 
-DEFAULT_SETTLE_TIMEOUT = 10
-DEFAULT_REVEAL_TIMEOUT = 3
+DEFAULT_REVEAL_TIMEOUT = 30
+DEFAULT_SETTLE_TIMEOUT = DEFAULT_REVEAL_TIMEOUT * 20
```

Comment:
Using larger values because I was testing with more than 3 hops (now we can use `* 20` hops with these defaults), the values choosen are arbitrary.